bigtop-packages/src/common/hadoop/do-component-build

```diff
@@ -125,6 +125,8 @@ MAVEN_OPTS+="-DskipTests -DskipTest -DskipITs "
 . $(dirname ${0})/maven_deploy.sh
 
 # Build artifacts
+# remove phantomjs dependency
+sed -i '/^phantomjs-prebuilt/,+12d;N;/phantomjs/!P;D' hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/yarn.lock
```

Comment:
Should we use `patch*.diff` file rather than ad hoc editing in the do-component-build? I think the sed scripting is possible to modify the file in unexpected way if the yarn.lock is updated (on Hadoop side).

Reply:
That should be a better way. I'll upload a new patchset. Thanks, @iwasakims